extensions/amp-consent/0.1/consent-info.js

```diff
@@ -251,6 +251,20 @@ export function convertValueToState(value) {
   return CONSENT_ITEM_STATE.UNKNOWN;
 }
 
+/**
+ * Helper function to convert response enum value to CONSENT_ITEM_STATE value
+ * @param {*} value
+ * @return {?CONSENT_ITEM_STATE}
+ */
+export function convertEnumValueToState(value) {
+  if (value === 'accepted') {
+    return CONSENT_ITEM_STATE.ACCEPTED;
+  } else if (value === 'rejected') {
+    return CONSENT_ITEM_STATE.REJECTED;
+  }
+  return null;
```

Comment:
one more comment. We need to add check to `unknown` since we fall back to `null` now : )